packages/docs/docs/auth/methods/index.md

```diff
@@ -1,48 +1,48 @@
 ---
-sidebar_position: 16
+sidebar_position: 1
 tags: [auth]
 ---
 
-# Auth Integration Patterns
+# Authentication Methods
```

Comment:
Like this cleanup

Reply:
Unfortunately, I do think something like "Patterns" is better here.  The reason is that methods can man other things as well such as "JWT", "Basic Auth" and the like.  This is more of what type of application is connecting.  Open to suggestions on the language but Methods I think is not quite right.

Reply:
Makes sense. I can maintain patterns